src/components/@shared/atoms/Lists/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ReactElement, ReactNode } from 'react'
+import styles from './index.module.css'
+
+export interface ListProps {
+  items: any[]
+  ordered?: boolean
+}
+
+export interface ListItemProps {
+  children?: ReactNode
+  ol?: boolean
+}
+
+export function ListItem({ children, ol }: ListItemProps): ReactElement {
+  const classes = ol
+    ? `${styles.item} ${styles.olItem}`
+    : `${styles.item} ${styles.ulItem}`
+
+  return (
+    <li className={classes}>
+      <span>{children}</span>
+    </li>
+  )
+}
+
+export function List({ items, ordered }: ListProps): ReactElement {
```

Comment:
mmmh, would not refactor this component just for storybook and this looks like you do just that which is a bad practice as now we are testing something which is actually not in use. If you want to create markup only for storybook, then please use their decorators https://storybook.js.org/docs/react/writing-stories/decorators